app/components/LatestTrailer/LatestTrailer.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  FlatList,
+  Image,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+  type ListRenderItemInfo,
+} from 'react-native';
+import { DropDown } from '..';
+import { Icons } from '../../assets';
+import { baseUrls, Strings } from '../../constants';
+import { styles } from './LatestTrailerStyles';
+import type { ItemType, LatestTrailerProp } from './LatestTrailerTypes';
+
+const LatestTrailer = ({
+  listTitle,
+  filterList,
+  movieList,
+}: LatestTrailerProp) => {
+  const listTextStyle = StyleSheet.flatten([
```

Comment:
do not create a local variable for style flatten used directly because when you can use a variable then it unnecessarily assigns memory on ROM.